src/main/java/org/apache/commons/cli/Options.java

```diff
@@ -54,6 +54,18 @@ public class Options implements Serializable {
     /** A map of the option groups */
     private final Map<String, OptionGroup> optionGroups = new LinkedHashMap<>();
 
+    /**
+     * Adds options to this option.
+     * @param options the options to add.
+     * @return this for chaining.
```

Comment:
We do not need to state the purpose of returning `this` as fluent style coding since the return value can be used for _anything_.